pallets/simple-crowdfund/src/lib.rs

```diff
@@ -1,269 +1,349 @@
-//! Simple Crowdfund Example
-//! - example of using `child-trie` in practice
-//! - designed to be a more simple version of polkadot/runtime/crowdfund
+//! Simple Crowdfund
+//!
+//! This pallet demonstrates a simple on-chain crowd-funding mechanism.
+//! It is based on Polkadot's crowdfund pallet, but is simplified and decoupled
+//! from the parachain logic.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
 use parity_scale_codec::{Decode, Encode};
-use sp_core::{Blake2Hasher, Hasher};
-use rstd::prelude::*;
+use sp_core::Hasher;
+use sp_std::prelude::*;
 use sp_runtime::{
-    traits::{AccountIdConversion, Saturating, Zero},
-    ModuleId,
+	traits::{AccountIdConversion, Saturating, Zero},
+	ModuleId,
```

Comment:
Are you intending to make these formatting changes?

Reply:
Yes, the old outdated version still had spaces for indents. We've been using tabs for a while, and recently (#253) codified that with `cargo fmt`.